scripts/events_module/condition_events.py

```diff
@@ -675,14 +675,15 @@ def handle_already_disabled(cat):
         for condition in conditions:
 
             # checking if the cat has a congenital condition to reveal and handling duration and death
+            prev_lives = game.clan.leader_lives
             status = cat.moon_skip_permanent_condition(condition)
 
             # if cat is dead, break
-            if cat.dead or cat.status == 'leader':
+            if cat.dead or game.clan.leader_lives < prev_lives:
                 triggered = True
                 event_types.append("birth_death")
                 event = f"{cat.name} died from complications caused by {condition}."
-                if cat.status == "leader" and not cat.dead:
+                if cat.status == "leader" and game.clan.leader_lives >= 1:
                     event = f"{cat.name} lost a live to {condition}."
```

Comment:
This isn't something new for the PR, but if you want to change it to "lost a life to" instead of "lost a live to" while you have this file open
I'd really appreciate it